### PR TITLE
[CHK-1856] fix: Make `getTransactionId` return transaction id encoded in base64

### PR DESCRIPTION
**Pull request by giovanniberti**
#### List of Changes

 * make `getTransactionId` API return transaction id encoded in base64

#### Motivation and Context

This change is required because of length limitation in NPG id fields

#### How Has This Been Tested?

Unit tests

#### Open points

* `PaymentMethodsService#uuidToBase64` is copied form `transactions-service`, I think it would be best to move it to `ecommerce-commons` (probably a method under `TransactionId`)

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

Depends on #97

### CODE REVIEW

**Comment by pietro-tota** [comment]
add a brief description for `List of Changes` and `Motivation and Context` changelog sections😄

File: src/test/java/it/pagopa/ecommerce/payment/methods/controller/PaymentMethodsControllerTests.java [hunk 1]
```diff
@@ -29,6 +29,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
```
Comment (pietro-tota) [inline src/test/java/it/pagopa/ecommerce/payment/methods/controller/PaymentMethodsControllerTests.java:32]: unused import?

**Review by pietro-tota — DISMISSED** [review]
Well done, just a comment

File: src/test/java/it/pagopa/ecommerce/payment/methods/controller/PaymentMethodsControllerTests.java (hunk 1 above)
Comment (giovanniberti) [reply src/test/java/it/pagopa/ecommerce/payment/methods/controller/PaymentMethodsControllerTests.java]: Fixed in https://github.com/pagopa/pagopa-ecommerce-payment-methods-service/pull/99/commits/6bf9a89ee8ac77be504430c2e470ca00269ebc6d

**Review by pietro-tota — DISMISSED** [review]

**Review by infantesimone — APPROVED** [review]

**Pull request merged**